src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -2033,6 +2033,14 @@ private List<CompletionResult> GetResultForIdentifier(CompletionContext completi
             result = CompletionCompleters.CompleteCommandArgument(completionContext);
```

Comment:
@vexx32, your last commit had 5 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.0.md.https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/ should work
```powershell
Failed to complete request to "https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.0.md.https://blogs.msdn.microsoft.com/dotnet/2016/09/26/introducing-net-standard/ should work
```powershell
Failed to complete request to "https://blogs.msdn.microsoft.com/dotnet/2016/09/26/introducing-net-standard/". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.0.md.https://blogs.msdn.microsoft.com/powershell/2017/01/31/powershell-open-source-community-dashboard/ should work
```powershell
Failed to complete request to "https://blogs.msdn.microsoft.com/powershell/2017/01/31/powershell-open-source-community-dashboard/". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/demos/WindowsPowerShellModules/README.md.https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/ should work
```powershell
Failed to complete request to "https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://blogs.msdn.microsoft.com/powershell/2017/01/31/powershell-open-source-community-dashboard/ should work
```powershell
Failed to complete request to "https://blogs.msdn.microsoft.com/powershell/2017/01/31/powershell-open-source-community-dashboard/". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Reply:
Unrelated to this PR. I blame Travis for irresponsibly including Markdown links that don't exist yet. 😝